### ISSUE TITLE
Do not override gtk-theme defines

### ISSUE THREAD

**Issue by raveit65** [rhinstaller/anaconda#1859]
With https://github.com/rhinstaller/anaconda/commit/3ea114e9e4a13eeb08016f1a43203d364768aa2d
from https://github.com/rhinstaller/anaconda/pull/1817
you override the very common define of gtk+ themes `theme_bg_color` with a own define.
```
/* theme colors/images */

@define-color theme_bg_color @fedora;
```
But the define `theme_bg_color` is used all over the css in themes.
The result is this.
![anaconda-30 25-2 fc30 x86_64-bad-from-fedora-mate_compiz-live-x86_64-30-20190220 n 0](https://user-images.githubusercontent.com/961604/53695546-4a873900-3dbd-11e9-8a0b-351c5d563907.png)

This is normal view with BlueMenta theme with anaconda-30.23-1.fc30
![anaconda-30 23-1 fc30 x86_64-good-from-fedora-mate_compiz-live-x86_64-rawhide-20190217 n 0](https://user-images.githubusercontent.com/961604/53695566-786c7d80-3dbd-11e9-96d7-d057ff86ae80.png)

As you can see font-colors aren't readable with a blue background

For the record:
You don't see that with adwaita theme because adwaita  use hex color values instead of color defines.

downstream report at https://bugzilla.redhat.com/show_bug.cgi?id=1684808

**Comment by poncovka** [comment]
Thanks for catching this bug!

**Comment by jkonecny12** [comment]
Resolved by b5b353d420d33c34f6bc56740ef5ae126829b83a
